src/Orders/Entities/GetOrdersResponse.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+namespace ArrowSphere\PublicApiClient\Orders\Entities;
+
+use ArrowSphere\PublicApiClient\AbstractEntity;
+use ArrowSphere\PublicApiClient\Entities\Pagination;
+
+class GetOrdersResponse extends AbstractEntity
+{
+    public const PAGINATION = 'pagination';
+    public const ORDERS = 'orders';
+
+    /**
+     * @var \ArrowSphere\PublicApiClient\Orders\Entities\Order[]
+     */
+    private $orders;
+
+    /**
+     * @var Pagination
+     */
+    private $pagination;
+
+    public function __construct(array $data)
+    {
+        parent::__construct($data['data']);
+
+        $this->orders = array_map(
+            static function (array $order) {
```

Comment:
Or synthetically:
`static fn (array $order) => new Order($order)`

Reply:
ok i think i will run rector to fix those kind of issues :100: